main/res/values-el/strings.xml

```diff
@@ -1994,7 +1998,13 @@
         <item quantity="other">Η διαδρομη φορτώθηκε (%d σημεία αναφοράς)</item>
     </plurals>
     <string name="load_track_error">Σφάλμα κατά την φόρτωση της διαδρομής</string>
+    <string name="map_clear_track">Εκκαθάριση διαδρομής</string>
+    <string name="map_clear_track_confirm">Θέλετε να αφαιρέσετε διαδρομή \"%s\";</string>
```

Comment:
Thats a strange one. I fixed the missing ? on Crowdin but it tried to convince me that is wrong:
![image](https://user-images.githubusercontent.com/949669/150699985-00eeb8c3-2225-4783-ba5c-af0316a681b3.png)

I changed it nonthelesse.